documentation/requirements/case_view.md

```diff
@@ -0,0 +1,20 @@
+# Case View
+The `CaseView` displays the data and interpretation for a case to the user.
+
+## Case data
+The data is shown in a table in which:
+- there is a row for each attribute having data in the case
+- there is a column for each date at which one or more test results were given.
```

Comment:
I'm wondering if we should make the tests "domain agnostic". E.g. refer to "values" rather than "test results" etc, or maybe keep the pathology domain tests separate and identified as such?